karma.conf.js

```diff
@@ -19,5 +19,6 @@ module.exports = function(config) {
     // Fail after timeout
     browserDisconnectTimeout: 100000,
     browserNoActivityTimeout: 100000,
+    karmaTypescriptConfig: { tsconfig: './tsconfig.json' },
```

Comment:
the key here is duplicate, see above line 9 it should be added there

Reply:
@ryanio fixed

Reply:
Oh sorry I think you misunderstood my comment, I meant the `karmaTypescriptConfig` key not the timeout

Reply:
@ryanio just fixed again, returned that line back and grouped `karmaTypescriptConfig` settings